audio-worklet/src/noisegate-script-processor.js

```diff
@@ -98,7 +106,7 @@ class NoiseGate {
   onaudioprocess_(event) {
     let inputBuffer = event.inputBuffer;
     let channel0 = inputBuffer.getChannelData(0);
-
+    
```

Comment:
Got it. I just installed JSLint which is noticing this error. For some reason, this error wasn't caught with the TrailingSpaces package.